persistence-model/src/test/scala/hmda/persistence/serialization/validation/SubmissionLarStatsProtobufSerializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.persistence.serialization.validation
+
+import hmda.persistence.messages.events.validation.SubmissionLarStatsEvents.{ MacroStatsUpdated, SubmittedLarsUpdated }
+import hmda.persistence.model.serialization.SubmissionLarStatsEvents.{ MacroStatsUpdatedMessage, SubmittedLarsUpdatedMessage }
+import org.scalacheck.Gen
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class SubmissionLarStatsProtobufSerializerSpec extends PropSpec with PropertyChecks with MustMatchers {
+  val serializer = new SubmissionLarStatsProtobufSerializer()
+
+  property("SubmittedLarsUpdated messages must be serialized to binary and back") {
+    forAll(Gen.choose(0, 100000)) { total =>
+      val msg = SubmittedLarsUpdated(totalSubmitted = total)
+      val bytes = serializer.toBinary(msg)
+      serializer.fromBinary(bytes, serializer.SubmittedLarsUpdatedManifest) mustBe msg
+    }
+  }
+
+  property("MacroStatsUpdated messages must be serialized to binary and back") {
+    val difGen = Gen.choose(-50000, 50000)
+    forAll(Gen.choose(0, 50000)) { num =>
+      val msg = MacroStatsUpdated(
+        totalValidated = num + difGen.sample.get,
```

Comment:
I'm not sure I see the value in using `num` as the base for each value. What's the difference between this strategy and just sampling `Gen.choose(0,50000)` for all of the different values in the object?